server/core/redis/redis_test.go

```diff
@@ -1,4 +1,4 @@
-package redis_test
+package redis
```

Comment:
This should remain as a separate package so we continue to test the exported surface of the `redis` package.

If you want to test internals, you should create a file called `redis_internal_test.go`, there are examples elsewhere in the code.